pages/add/form/get-help-fields.js

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import validator from './validator';
 import Service from '../../../js/service';
 
-let HelpTypes = React.createClass({
-  getInitialState() {
-    return {
+class HelpTypes extends React.Component {
+  constructor() {
```

Comment:
This is a choice. I prefer to acknowledge that the constructor is supposed to take `props`, and let the code figure out what to do based on the fact that it's getting `super(props)` called, concretely, as `super(undefined)` when we don't have a properset in the JSX that creates the object.

You can leave `props` off, but I'd prefer not to.